unitary/alpha/qudit_effects.py

```diff
@@ -127,3 +125,27 @@ class QuditFlip(Flip):
 
     def __init__(self, dimension: int, state0: int, state1: int):
         super().__init__(state0=state0, state1=state1)
+
+
+class Superpose(QuantumEffect):
```

Comment:
Let's move this into unitary.alpha.qubit_effects.Superposition if possible.

The feedback from test users is that no one wants to worry about Qudits vs Qubits, and that we should be able to do Superpostion() and it should work on whatever dimension (which the qudit_gate you wrote already supports)

Reply:
Thanks Doug. It's combined into qubit_effects.Superposition.